include/swift/AST/ModuleDependencies.h

```diff
@@ -1042,8 +1043,9 @@ class SwiftDependencyScanningService {
   /// If use clang include tree.
   bool UseClangIncludeTree = false;
 
-  /// CAS ObjectStore Instance.
+  /// CAS Instance.
   std::shared_ptr<llvm::cas::ObjectStore> CAS;
+  std::shared_ptr<llvm::cas::ActionCache> Cache;
```

Comment:
Given the extensive use of the term Cache in the scanner, could we please give this a more-descriptive name?